plugins/woocommerce-admin/client/customer-effort-score-tracks/data/reducer.js

```diff
@@ -62,6 +63,16 @@ const reducer = ( state = DEFAULT_STATE, action ) => {
 				...state,
 				queue: [ ...state.queue, newTrack ],
 			};
+		case TYPES.SHOW_PRODUCT_MVP_FEEDBACK_MODAL:
```

Comment:
I concur with you. I followed the pattern we were using to not add more complexity to this part, but I think that it would be a really nice refactor to handle the visibility of the modals differently.